romancal/datamodels/schemas/image_files/image.schema.yaml

```diff
@@ -0,0 +1,327 @@
+              type: number
+            cdelt2:
+              title: second axis increment per pixel
+              type: number
+            cdelt3:
+              title: third axis increment per pixel
+              type: number
+            pc1_1:
+              title: linear transformation matrix element
+              type: number
+            pc1_2:
+              title: linear transformation matrix element
+              type: number
+            pc1_3:
+              title: linear transformation matrix element
+              type: number
+            pc2_1:
+              title: linear transformation matrix element
+              type: number
+            pc2_2:
+              title: linear transformation matrix element
+              type: number
+            pc2_3:
+              title: linear transformation matrix element
+              type: number
+            pc3_1:
+              title: linear transformation matrix element
+              type: number
+            pc3_2:
+              title: linear transformation matrix element
+              type: number
+            pc3_3:
+              title: linear transformation matrix element
+              type: number
+            ps3_0:
+              title: Coordinate table extension name
+              type: string
+            ps3_1:
+              title: Coordinate table column name
+              type: string
+            s_region:
+              title: spatial extent of the observation
+              type: string
+              default: " "
+            waverange_start:
+              title: lower bound of the default wavelength range
+              type: number
+            waverange_end:
+              title: upper bound of the default wavelength range
+              type: number
+            dispersion_direction:
+              title: Dispersion direction
+              type: integer
+            spectral_order:
+              title: Spectral order number
+              type: integer
+            v2_ref:
+              title: "[arcsec] Telescope v2 coordinate of the reference point"
+              type: number
+            v3_ref:
+              title: "[arcsec] Telescope v3 coordinate of the reference point"
+              type: number
+            vparity:
+              title: Relative sense of rotation between Ideal xy and V2V3
+              type: integer
+            v3yangle:
+              title: "[deg] Angle from V3 axis to Ideal y axis"
+              type: number
+            ra_ref:
+              title: "[deg] Right ascension of the reference point"
+              type: number
+            dec_ref:
+              title: "[deg] Declination of the reference point"
+              type: number
+            roll_ref:
+              title: "[deg] V3 roll angle at the ref point (N over E)"
+              type: number
+            velosys:
+              title: "[m/s] Radial velocity wrt Barycenter"
+              type: number
+            specsys:
```

Comment:
remove this and everything else to the end of wcsinfo

Reply:
removed